packages/tasks/src/model-libraries.ts

```diff
@@ -435,6 +435,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/jbloomAus/SAELens",
 		filter: false,
 	},
+	"sam": {
```

Comment:
This would impact lots of models https://huggingface.co/models?other=sam , including SAM1 release

Should we use `sam2` here?